README.md

```diff
@@ -36,8 +36,8 @@ To set up Cloudfront integration, you need to
 1. Create the required resources in your AWS infrastructure — a secret stored in the Secrets Manager, a CloudFormation stack, and a CloudFront distribution.
 2. Configure the Fingerprint Pro JS Agent on your site to communicate with your created Lambda@Edge function using the [endpoint](https://dev.fingerprint.com/docs/js-agent#endpoint) parameter.
 
-See [Cloudflare Proxy Integration guide](https://dev.fingerprint.com/docs/cloudfront-proxy-integration) in our documentation for step-by-step instructions. If you have any questions, reach out to our [support team](https://fingerprint.com/support/). 
+See [CloudFront Proxy Integration guide](https://dev.fingerprint.com/docs/cloudfront-proxy-integration) in our documentation for step-by-step instructions. If you have any questions, reach out to our [support team](https://fingerprint.com/support/). 
```

Comment:
These changes are not supposed to be here, am I right?

Reply:
As discussed on slack, it's just to keep dev and main in sync.